crates/trippy/Cargo.toml

```diff
@@ -53,8 +53,14 @@ clap_mangen = "0.2.20"
 [dev-dependencies]
 test-case.workspace = true
 serde_yaml.workspace = true
+insta = { version = "1.39.0", features = ["serde"] }
 pretty_assertions = "1.4.0"
 
+# Improve testing speed https://insta.rs/docs/quickstart/#optional-faster-runs
+[profile.dev.package]
+insta.opt-level = 3
+similar.opt-level = 3
+
```

Comment:
Gives:

```
warning: profiles for the non root package will be ignored, specify profiles at the workspace root:
package:   /Users/tristan/projects/trippy/crates/trippy/Cargo.toml
workspace: /Users/tristan/projects/trippy/Cargo.toml
```

I'd suggest we remove this, I doubt the speedup is significant for the small file sizes.